component/Header.tsx

```diff
@@ -1,15 +1,15 @@
-import getConfig from 'next/config'
 import Image from 'next/image'
-import { useRouter } from 'next/router'
 import React from 'react'
+import getConfig from 'next/config'
+import { useRouter } from 'next/router'
 
 const { publicRuntimeConfig } = getConfig()
 
-const Header = () => {
+const Header = ({fixedPosition = true}) => {
     const toDownloadLink = () => window.open(publicRuntimeConfig.DOWNLOAD_BETTERSOCIAL_APP_URL, '_blank')
     
     return (
-        <div className="flex w-full md:max-w-M lg:max-w-M xl:max-w-M justify-between items-center min-h-[45px] fixed top-0 p-4">
+        <div className={`flex w-full md:max-w-M lg:max-w-M xl:max-w-M justify-between items-center min-h-[45px] top-0 p-4 ${fixedPosition ? 'fixed' : ''}`}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `onClick` handler for the `Image` component is not wrapped in a function. This could lead to the `toDownloadLink` function being called immediately when the component renders, rather than when the image is clicked. To fix this, wrap the `onClick` handler in an anonymous function.
```diff
- <Image onClick={() => toDownloadLink()} alt="logo helio" src="/image/LogoBrand.svg" width={60} height={28} />
+ <Image onClick={() => {toDownloadLink()}} alt="logo helio" src="/image/LogoBrand.svg" width={60} height={28} />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->